src/components/Button.js

```diff
@@ -248,4 +249,4 @@ class Button extends React.Component {
   };
 }
 
-module.exports = Radium(Button);
+module.exports = Button;
```

Comment:
Do you think that removing `Radium` will fix the bug you are experiencing?

Reply:
i do